plugins/appLoad.ts

```diff
@@ -86,5 +86,12 @@ const listenGlobalKeyboardEvents = (app, store) => {
 }
 
 export default ({ app, store }): void => {
+  // Bind Global Keyboard Events
   listenGlobalKeyboardEvents(app, store)
+
+  // Fetch User balance
+  const account = store.getters.getAuthAddress
```

Comment:
???? 
Please use separated file

Reply:
ok so what we can do is polling after every 5 to 10 seconds because user can add balance to wallet and this needs to be reflected on our app also.

Reply:
```
api.derive.balances.all(accountId, ({ availableBalance }) => { // set balance on change })
```

@vikiival I think this should work, right ??

Reply:
Yup, but you have to unsubscribe